packages/dendron-cli/src/commands/notes.ts

```diff
@@ -145,6 +157,23 @@ export class NoteCLICommand extends CLICommand<CommandOpts, CommandOutput> {
           this.print(`deleted ${note.fname}`);
           return { data: { payload: note.fname, rawData: resp } };
         }
+        case NoteCommands.MOVE:
+        case NoteCommands.RENAME: {
```

Comment:
semantically, rename is a convenience function on top of `move` that executes `move` while keeping the vault name the same. this CLI argument doesn't actually do that (you can change the vault name using a flag). 
I would recommend simplifying this PR to just add the `move` command since its a superset of the functionality (and because we prefer for a PR to introduce one change)